### PR TITLE
l10n: Correct spelling

### PR DESCRIPTION
**Pull request by Valdnet**
Reported by the user from Transifex.

Signed-off-by: Valdnet <47037905+Valdnet@users.noreply.github.com>

### CODE REVIEW

**Comment by dartcafe** [comment]
@Valdnet Great, thanks. 👍  According to #2057 we should use the present perfect here.

File: lib/Service/ActivityService.php [hunk 1]
```diff
@@ -355,7 +355,7 @@ private function getMessageFiltered() {
 
 				return $this->userIsActor
 					? $this->trans->t('You deleted {sharee}\'s share')
-					: $this->trans->t('{actor} deleted a share');
```
Comment (dartcafe) [inline lib/Service/ActivityService.php:358]: Avoiding the sharee here was by intention, for possible privacy concerns.

**Review by dartcafe — CHANGES_REQUESTED** [review]

File: lib/Service/ActivityService.php (hunk 1 above)
Comment (Valdnet) [reply lib/Service/ActivityService.php]: I have restored the text string 😉.

**Review by dartcafe — APPROVED** [review]

File: lib/Service/ActivityService.php [hunk 2]
```diff
@@ -187,7 +187,7 @@ private function getMessageFull() {
 			case ShareEvent::REGISTRATION:
 				return $this->userIsActor
 					? $this->trans->t('You registered to poll {pollTitle}')
-					: $this->trans->t('{sharee} registered to poll {pollTitle}');
+					: $this->trans->t('{actor} registered to poll {pollTitle}');
```
Comment (dartcafe) [inline lib/Service/ActivityService.php:190]: In this case {sharee} was right to fetch the correct user type, when displaying the user's name

**Review by dartcafe — CHANGES_REQUESTED** [review]

**Comment by dartcafe** [comment]
@Valdnet 😆 Too fast. The review approvement was related to the change. I will change the wordings in the next hour and fix the last change request in a another PR. Thanskf or review and correcting.

**Pull request merged**